mathesar/tests/api/test_table_api.py

```diff
@@ -44,6 +44,19 @@ def missing_keys_json_data_file():
     return data_file
 
 
+@pytest.fixture
+def patents_excel_data_file():
+    data_filepath = 'mathesar/tests/data/patents.xlsx'
```

Comment:
Use the `patents_excel_filepath` fixture here.

Reply:
Done

Reply:
you can pass in `patents_excel_filepath` as a param to the `patents_excel_data_file`.

Reply:
Done